packages/jest-preset-mc-app/load-config.js

```diff
@@ -0,0 +1,42 @@
+const { cosmiconfigSync } = require('cosmiconfig');
+
+const moduleName = 'jest-mc-app';
+const explorer = cosmiconfigSync(moduleName);
+
+const defaultConfig = {
+  silenceConsoleWarnings: [
+    /.*Warning: componentWillReceiveProps has been renamed.*/,
+    /.*CellMeasurerCache should only measure a cell's width or height.*/,
+  ],
+  notThrowWarnings: [
+    /.*@commercetools-frontend\/permissions.*/,
+    /.*Warning: React.createFactory() is deprecated.*/,
+  ],
+};
+
+const mergeSilenceConsoleWarnings = createListMergerWithDefaults(
+  'silenceConsoleWarnings'
+);
+const mergeNotThrowWarnings = createListMergerWithDefaults('notThrowWarnings');
+
+const loadConfig = () => {
+  const configFile = explorer.load(pathToConfig);
+  if (jestConfig.isEmpty) {
+    return defaultConfig;
+  }
+  const customConfig = configFile.config;
+  return {
+    ...defaultConfig,
+    ...customConfig,
+    ...mergeSilenceConsoleWarnings(customConfig),
+    ...mergeNotThrowWarnings(customConfig),
```

Comment:
These functions basically merge the default values with custom ones.
I'm not 100% sure about this though, if we should just let the user "override" the field.

Reply:
I think it is a flat structure after all. Maybe it is just fine to keep it simple for now. Deep merging would be a non-breaking change. Removing it would be. So maybe it is fine as is.